src/PrestaShopBundle/Form/Admin/Type/TypeaheadProductCollectionType.php

```diff
@@ -90,11 +91,24 @@ public function buildView(FormView $view, FormInterface $form, array $options)
                         break;
 
                     default:
+                        $lang = Context::getContext()->language->id;
                         $product = $this->productAdapter->getProduct($id);
+
+                        $productName = '';
+                        if (!empty($product->name[$lang])) {
+                            $productName .= $product->name[$lang] . ' ';
+                        }
```

Comment:
Fallback on `reset($product->name)`?

Reply:
All right, added the fallback if the name is not found in the current language ! 👍